parts/linux/cloud-init/artifacts/components.json

```diff
@@ -381,6 +381,29 @@
       "multiArchVersions": [
         "v0.7.1"
       ]
+    },
+    {
+      "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:*",
+      "amd64OnlyVersions": [],
+      "multiArchVersions": [
```

Comment:
I know @Devinwong is working on a new schema, should we wait for that to be merged and then realign?

Reply:
yep! I'll close this one.